app/src/test/java/com/codesoom/assignment/application/ProductServiceTest.java

```diff
@@ -0,0 +1,290 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class ProductServiceTest {
+
+    private static final String PRODUCT_NAME = "test";
+    private static final String CREATE_POSTFIX = "...";
+    private static final String UPDATE_POSTFIX = "!!!";
+
+    private ProductService productService;
+    private ProductRepository productRepository;
+
+    @BeforeEach
+    void setUp() {
+        productRepository = mock(ProductRepository.class);
+        // subject
+        productService = new ProductService(productRepository);
+    }
+
+    private List<Product> createProducts() {
+        List<Product> products = new ArrayList<>();
+
+        Product product = new Product();
+        product.setMaker(PRODUCT_NAME);
```

Comment:
```suggestion
        Product product = createProduct();
```

여기는 `createProduct`를 사용하면 좋을 것 같아요